### ISSUE TITLE
[Controller Request]: 

### ISSUE THREAD

**Issue by cuppajojo** [ndeadly/MissionControl#545]
### Controller Name

X-Plorer (Guitar Hero Xbox Controller)

### Current Controller Behaviour

Controller is not successful in pairing with the console

### Supported Features

- [ ] Rumble/Vibration
- [ ] Motion Controls
- [ ] Player Number Indicator
- [ ] Battery Level Reporting

### Additional Details

It's just the xbox 360 Guitar Hero Guitar.

**Comment by ndeadly** [comment]
Xbox 360 controllers do not use Bluetooth and require the USB wireless Adapter to work outside of Xbox consoles. This can only be supported once Mission Control supports USB.

**Comment by Ritaku02** [comment]
Any idea when you'll have USB support? I tried sys-con, but it has issues with my Switch on latest firmware

**Comment by ndeadly** [comment]
Not really. I have it partially done but have bumped into an issue I don't know how to solve so I've benched it for now while I work on other things.

**Comment by Ritaku02** [comment]
Okies, no prob! Thanks for the answer :3
